src/index.ts

```diff
@@ -1,5 +1,129 @@
-export class Hello {
-  public sayHello() {
-    return 'hello, world!';
+import { Duration, RemovalPolicy } from 'aws-cdk-lib';
+import { BackupProps, ClusterParameterGroup, DatabaseCluster, DatabaseSecret } from 'aws-cdk-lib/aws-docdb';
+import {
+  InstanceClass,
+  InstanceSize,
+  InstanceType,
+  ISecurityGroup,
+  IVpc,
+  SecurityGroup,
+  SubnetType,
+} from 'aws-cdk-lib/aws-ec2';
+import { IKey, Key } from 'aws-cdk-lib/aws-kms';
+import { Construct } from 'constructs';
+import { Namer } from 'multi-convention-namer';
+
+export interface DocumentDbProps {
```

Comment:
Nit: I know this is not a universal convention, but I think we should be consistent with the AWS naming and use `DocumentDB` instead of `DocumentDb`